core/conversion/var/Var.cpp

```diff
@@ -21,6 +21,28 @@ Var::Var(nvinfer1::ITensor* p) : type_(Type::kITensor) {
   ptr_.tensor = p;
 }
 
+Var::IValueType Var::determineIValueType(torch::jit::IValue* p) {
```

Comment:
@peri044 you can implement a macro to do this similar to what we have in var_inl.h

Reply:
Vs duplicating PyTorch's type system

Reply:
Made changes now to rely on pytorch's type system